[2022-06-11 10:22:31,306] [builder.py:185]            [INFO]     |====================================================================================================|
[2022-06-11 10:22:31,307] [builder.py:185]            [INFO]     |                               Trakt Recommended Collection in Movies                               |
[2022-06-11 10:22:31,307] [builder.py:185]            [INFO]     |====================================================================================================|
[2022-06-11 10:22:31,307] [builder.py:186]            [INFO]     |                                                                                                    |
[2022-06-11 10:22:31,308] [builder.py:191]            [INFO]     |============================= Validating Trakt Recommended Attributes ==============================|
[2022-06-11 10:22:31,308] [builder.py:212]            [DEBUG]    |                                                                                                    |
[2022-06-11 10:22:31,309] [builder.py:213]            [DEBUG]    | Validating Method: template                                                                        |
[2022-06-11 10:22:31,309] [meta.py:103]               [DEBUG]    | Value: [{'value': ['recommended'], 'custom': ['recommended'], 'key_name': 'Recommended', 'key': 'recommended', 'name': 'trakt'}, {'value': ['recommended'], 'custom': ['recommended'], 'key_name': 'Recommended', 'key': 'recommended', 'name': 'customsync'}, {'value': ['recommended'], 'custom': ['recommended'], 'key_name': 'Recommended', 'key': 'recommended', 'name': 'chart_shared'}] |
[2022-06-11 10:22:31,313] [builder.py:437]            [DEBUG]    |                                                                                                    |
[2022-06-11 10:22:31,313] [builder.py:438]            [DEBUG]    | Validating Method: sync_mode                                                                       |
[2022-06-11 10:22:31,313] [builder.py:442]            [DEBUG]    | Value: sync                                                                                        |
[2022-06-11 10:22:31,314] [builder.py:582]            [DEBUG]    |                                                                                                    |
[2022-06-11 10:22:31,314] [builder.py:583]            [DEBUG]    | Validating Method: collection_order                                                                |
[2022-06-11 10:22:31,314] [builder.py:584]            [DEBUG]    | Value: custom                                                                                      |
[2022-06-11 10:22:31,314] [builder.py:600]            [DEBUG]    |                                                                                                    |
[2022-06-11 10:22:31,315] [builder.py:604]            [DEBUG]    | Validating Method: label                                                                           |
[2022-06-11 10:22:31,315] [builder.py:605]            [DEBUG]    | Value: Trakt                                                                                       |
[2022-06-11 10:22:31,315] [builder.py:600]            [DEBUG]    |                                                                                                    |
[2022-06-11 10:22:31,316] [builder.py:604]            [DEBUG]    | Validating Method: trakt_chart                                                                     |
[2022-06-11 10:22:31,316] [builder.py:605]            [DEBUG]    | Value: {'chart': 'recommended', 'limit': 100}                                                      |
[2022-06-11 10:22:31,316] [builder.py:600]            [DEBUG]    |                                                                                                    |
[2022-06-11 10:22:31,317] [builder.py:604]            [DEBUG]    | Validating Method: summary                                                                         |
[2022-06-11 10:22:31,317] [builder.py:605]            [DEBUG]    | Value: A collection of Trakt's Weekly movie List                                                   |
[2022-06-11 10:22:31,317] [builder.py:600]            [DEBUG]    |                                                                                                    |
[2022-06-11 10:22:31,318] [builder.py:604]            [DEBUG]    | Validating Method: sort_title                                                                      |
[2022-06-11 10:22:31,318] [builder.py:605]            [DEBUG]    | Value: !01_Trakt Recommended                                                                       |
[2022-06-11 10:22:31,318] [builder.py:600]            [DEBUG]    |                                                                                                    |
[2022-06-11 10:22:31,319] [builder.py:604]            [DEBUG]    | Validating Method: url_poster                                                                      |
[2022-06-11 10:22:31,319] [builder.py:605]            [DEBUG]    | Value: https://raw.githubusercontent.com/meisnate12/Plex-Meta-Manager-Images/master/chart/Trakt%20Recommended.jpg |
[2022-06-11 10:22:34,552] [plex.py:742]               [DEBUG]    |                                                                                                    |
[2022-06-11 10:22:34,557] [plex.py:745]               [DEBUG]    | Looking for: Trakt Recommended                                                                     |
[2022-06-11 10:22:34,557] [builder.py:764]            [INFO]     |                                                                                                    |
[2022-06-11 10:22:34,558] [builder.py:765]            [INFO]     | Validation Successful                                                                              |
[2022-06-11 10:22:34,558] [plex_meta_manager.py:539]  [INFO]     |                                                                                                    |
[2022-06-11 10:22:34,559] [plex_meta_manager.py:541]  [INFO]     |=============================== Running Trakt Recommended Collection ===============================|
[2022-06-11 10:22:34,559] [plex_meta_manager.py:553]  [INFO]     |                                                                                                    |
[2022-06-11 10:22:34,562] [plex_meta_manager.py:554]  [INFO]     | Sync Mode: sync                                                                                    |
[2022-06-11 10:22:34,562] [plex_meta_manager.py:557]  [DEBUG]    |                                                                                                    |
[2022-06-11 10:22:34,563] [plex_meta_manager.py:558]  [DEBUG]    | Builder: trakt_chart: {'chart': 'recommended', 'limit': 100, 'time_period': None}                  |
[2022-06-11 10:22:34,563] [plex_meta_manager.py:559]  [INFO]     |                                                                                                    |
[2022-06-11 10:22:34,563] [trakt.py:490]              [INFO]     | Processing Trakt Chart: 100 Recommended Movies                                                     |
[2022-06-11 10:22:36,635] [builder.py:1507]           [DEBUG]    |                                                                                                    |
[2022-06-11 10:22:36,636] [builder.py:1508]           [DEBUG]    | 100 IDs Found: [(361743, 'tmdb'), (545611, 'tmdb'), (414906, 'tmdb'), (338953, 'tmdb'), (293660, 'tmdb'), (639933, 'tmdb'), (634649, 'tmdb'), (453395, 'tmdb'), (648579, 'tmdb'), (157336, 'tmdb'), (680, 'tmdb'), (550, 'tmdb'), (507086, 'tmdb'), (957258, 'tmdb'), (155, 'tmdb'), (120, 'tmdb'), (496243, 'tmdb'), (744, 'tmdb'), (438631, 'tmdb'), (603, 'tmdb'), (324857, 'tmdb'), (546554, 'tmdb'), (18, 'tmdb'), (27205, 'tmdb'), (37165, 'tmdb'), (10681, 'tmdb'), (238, 'tmdb'), (329, 'tmdb'), (831946, 'tmdb'), (705861, 'tmdb'), (670, 'tmdb'), (752, 'tmdb'), (278, 'tmdb'), (78, 'tmdb'), (489, 'tmdb'), (13, 'tmdb'), (526896, 'tmdb'), (809140, 'tmdb'), (348, 'tmdb'), (244786, 'tmdb'), (299534, 'tmdb'), (752623, 'tmdb'), (324786, 'tmdb'), (68718, 'tmdb'), (17654, 'tmdb'), (11, 'tmdb'), (11324, 'tmdb'), (245891, 'tmdb'), (961484, 'tmdb'), (675353, 'tmdb'), (615457, 'tmdb'), (77, 'tmdb'), (280, 'tmdb'), (106646, 'tmdb'), (118340, 'tmdb'), (299536, 'tmdb'), (335787, 'tmdb'), (1124, 'tmdb'), (205596, 'tmdb'), (170, 'tmdb'), (807, 'tmdb'), (671, 'tmdb'), (335984, 'tmdb'), (76341, 'tmdb'), (629, 'tmdb'), (420821, 'tmdb'), (135397, 'tmdb'), (818397, 'tmdb'), (846250, 'tmdb'), (100402, 'tmdb'), (286217, 'tmdb'), (38, 'tmdb'), (185, 'tmdb'), (16869, 'tmdb'), (396535, 'tmdb'), (122, 'tmdb'), (530915, 'tmdb'), (24428, 'tmdb'), (333339, 'tmdb'), (679, 'tmdb'), (121, 'tmdb'), (769, 'tmdb'), (370172, 'tmdb'), (629542, 'tmdb'), (22, 'tmdb'), (530385, 'tmdb'), (646380, 'tmdb'), (98, 'tmdb'), (263115, 'tmdb'), (764, 'tmdb'), (70, 'tmdb'), (743563, 'tmdb'), (137, 'tmdb'), (6977, 'tmdb'), (577922, 'tmdb'), (374720, 'tmdb'), (419430, 'tmdb'), (637, 'tmdb'), (73, 'tmdb'), (11321, 'tmdb')] |
[2022-06-11 10:22:36,637] [builder.py:1509]           [DEBUG]    |                                                                                                    |
[2022-06-11 10:22:36,679] [builder.py:2009]           [INFO]     |                                                                                                    |
[2022-06-11 10:22:36,683] [builder.py:2010]           [INFO]     |============================== Adding to Trakt Recommended Collection ==============================|
[2022-06-11 10:22:36,685] [builder.py:2011]           [INFO]     |                                                                                                    |
[2022-06-11 10:22:36,689] [builder.py:2025]           [INFO]     |  1/38 | Trakt Recommended Collection | + | Pulp Fiction (1994)                                     |
[2022-06-11 10:28:11,882] [builder.py:2025]           [INFO]     |  2/38 | Trakt Recommended Collection | + | Fight Club (1999)                                       |
[2022-06-11 10:28:36,647] [builder.py:2025]           [INFO]     |  3/38 | Trakt Recommended Collection | + | The Dark Knight (2008)                                  |
[2022-06-11 10:28:48,191] [builder.py:2025]           [INFO]     |  4/38 | Trakt Recommended Collection | + | The Lord of the Rings: The Fellowship of the Ring (2001) |
[2022-06-11 10:28:54,289] [builder.py:2025]           [INFO]     |  5/38 | Trakt Recommended Collection | + | The Matrix (1999)                                       |
[2022-06-11 10:28:58,580] [builder.py:2025]           [INFO]     |  6/38 | Trakt Recommended Collection | + | Spider-Man: Into the Spider-Verse (2018)                |
[2022-06-11 10:29:58,657] [plex_meta_manager.py:668]  [DEBUG]    | Traceback (most recent call last):                                                                 |
                                                                 |   File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 449, in _make_request
                                                                 |     six.raise_from(e, None)
                                                                 |   File "<string>", line 3, in raise_from
                                                                 |   File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 444, in _make_request
                                                                 |     httplib_response = conn.getresponse()
                                                                 |   File "/usr/lib/python3.8/http/client.py", line 1386, in getresponse
                                                                 |     response.begin()
                                                                 |   File "/usr/lib/python3.8/http/client.py", line 312, in begin
                                                                 |     version, status, reason = self._read_status()
                                                                 |   File "/usr/lib/python3.8/http/client.py", line 273, in _read_status
                                                                 |     line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
                                                                 |   File "/usr/lib/python3.8/socket.py", line 669, in readinto
                                                                 |     return self._sock.recv_into(b)
                                                                 | socket.timeout: timed out
                                                                 | 
                                                                 | During handling of the above exception, another exception occurred:
                                                                 | 
                                                                 | Traceback (most recent call last):
                                                                 |   File "/usr/lib/python3.8/site-packages/requests/adapters.py", line 440, in send
                                                                 |     resp = conn.urlopen(
                                                                 |   File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 785, in urlopen
                                                                 |     retries = retries.increment(
                                                                 |   File "/usr/lib/python3.8/site-packages/urllib3/util/retry.py", line 550, in increment
                                                                 |     raise six.reraise(type(error), error, _stacktrace)
                                                                 |   File "/usr/lib/python3.8/site-packages/urllib3/packages/six.py", line 770, in reraise
                                                                 |     raise value
                                                                 |   File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 703, in urlopen
                                                                 |     httplib_response = self._make_request(
                                                                 |   File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 451, in _make_request
                                                                 |     self._raise_timeout(err=e, url=url, timeout_value=read_timeout)
                                                                 |   File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 340, in _raise_timeout
                                                                 |     raise ReadTimeoutError(
                                                                 | urllib3.exceptions.ReadTimeoutError: HTTPConnectionPool(redacted): Read timed out. (read timeout=60)
                                                                 | 
                                                                 | During handling of the above exception, another exception occurred:
                                                                 | 
                                                                 | Traceback (most recent call last):
                                                                 |   File "plex_meta_manager.py", line 570, in run_collection
                                                                 |     items_added, items_unchanged = builder.add_to_collection()
                                                                 |   File "/root/Plex-Meta-Manager/modules/builder.py", line 2032, in add_to_collection
                                                                 |     self.library.alter_collection(item, name, smart_label_collection=self.smart_label_collection)
                                                                 |   File "/root/Plex-Meta-Manager/modules/plex.py", line 650, in alter_collection
                                                                 |     field = next((f for f in item.fields if f.name == "collection"), None)
                                                                 |   File "/usr/lib/python3.8/site-packages/plexapi/base.py", line 483, in __getattribute__
                                                                 |     self._reload(_overwriteNone=False)
                                                                 |   File "/usr/lib/python3.8/site-packages/plexapi/base.py", line 357, in _reload
                                                                 |     data = self._server.query(key)
                                                                 |   File "/usr/lib/python3.8/site-packages/plexapi/server.py", line 696, in query
                                                                 |     response = method(url, headers=headers, timeout=timeout, **kwargs)
                                                                 |   File "/usr/lib/python3.8/site-packages/requests/sessions.py", line 542, in get
                                                                 |     return self.request('GET', url, **kwargs)
                                                                 |   File "/usr/lib/python3.8/site-packages/requests/sessions.py", line 529, in request
                                                                 |     resp = self.send(prep, **send_kwargs)
                                                                 |   File "/usr/lib/python3.8/site-packages/requests/sessions.py", line 645, in send
                                                                 |     r = adapter.send(request, **kwargs)
                                                                 |   File "/usr/lib/python3.8/site-packages/requests/adapters.py", line 532, in send
                                                                 |     raise ReadTimeout(e, request=request)
                                                                 | requests.exceptions.ReadTimeout: HTTPConnectionPool(redacted): Read timed out. (read timeout=60)
                                                                 | 
[2022-06-11 10:29:58,688] [plex_meta_manager.py:669]  [ERROR]    | Unknown Error: HTTPConnectionPool(redacted): Read timed out. (read timeout=60)
[2022-06-11 10:29:58,689] [plex_meta_manager.py:674]  [INFO]     | 
[2022-06-11 10:29:58,689] [plex_meta_manager.py:675]  [INFO]     |====================================================================================================|
[2022-06-11 10:29:58,690] [plex_meta_manager.py:675]  [INFO]     |                               Finished Trakt Recommended Collection                                |
[2022-06-11 10:29:58,691] [plex_meta_manager.py:675]  [INFO]     |                                    Collection Run Time: 0:07:27                                    |
[2022-06-11 10:29:58,691] [plex_meta_manager.py:675]  [INFO]     |====================================================================================================|
